src/ExportEdiJobs/ExportEdiJobsFilters/ExportMethodFilter/ExportMethodFilter.js

```diff
@@ -7,8 +8,10 @@ import {
 
 import { useConfigs } from './useConfigs';
 
-const ExportMethodFilter = (props) => {
-  const { configs, isFetching } = useConfigs();
+const ExportMethodFilter = ({ vendorId, ...rest }) => {
+  const { configs, isFetching, refetch } = useConfigs(vendorId);
+
+  useEffect(() => refetch(), [vendorId, refetch]);
```

Comment:
What if you add `organizationId` as react-query `key` in `useConfigs`, do you still need to use this `useEffect`, `refetch` and set `{ enabled: false }`?

Reply:
should work as well, thanx)